addons/website_sale_stock/models/product_template.py

```diff
@@ -27,28 +27,30 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
         if combination_info['product_id']:
             product = self.env['product.product'].sudo().browse(combination_info['product_id'])
             website = self.env['website'].get_current_website()
-            virtual_available = product.with_context(warehouse=website.warehouse_id.id).virtual_available
+            product_with_context = product.with_context(warehouse=website.warehouse_id.id)
+            qty_available = product_with_context.qty_available
+            qty_forecasted = product_with_context.incoming_qty
             combination_info.update({
-                'virtual_available': virtual_available,
-                'virtual_available_formatted': self.env['ir.qweb.field.float'].value_to_html(virtual_available, {'precision': 0}),
+                'qty_available': qty_available,
```

Comment:
Shouldn't this be `free_qty` for avoiding reserved quantities to be taken into account?